src/ameba.cr

```diff
@@ -40,4 +32,18 @@ module Ameba
   def run(config = Config.load)
     Runner.new(config).run
   end
+
+  macro ecr_supported?(&)
+    {% if compare_versions(Crystal::VERSION, "1.15.0") >= 0 %}
+      {{ yield }}
+    {% end %}
+  end
```

Comment:
Why not simply?

```suggestion
  macro ecr_supported?
    {{ compare_versions(Crystal::VERSION, "1.15.0") >= 0 }}
  end
```

... followed by `if Ameba.ecr_supported?` in the call-sites?

Reply:
My bad, nevermind that.